homeassistant/components/light/strings.json

```diff
@@ -477,20 +477,27 @@
   },
   "title": "Light",
   "triggers": {
-    "state": {
-      "description": "When the state of a light changes, such as turning on or off.",
-      "description_configured": "When the state of a light changes",
+    "turned_off": {
+      "description": "Triggers when a light is turned off.",
+      "description_configured": "Triggers when a light is turned off",
       "fields": {
         "behavior": {
-          "description": "The behavior of the targeted entities to trigger on.",
+          "description": "The behavior of the targeted lights to trigger on.",
           "name": "Behavior"
```

Comment:
The user will see a select named "Behavior" with the description "The behavior of the targeted lights to trigger on." and options "Any", "First", "Last".
I think this is really difficult to understand.

Maybe it would be clearer with: "Trigger when lights are turned off" / "Trigger when lights are turned on"?

Even better would be if we could have the description change depending on the select: "Trigger when all lights are turned off" / "Trigger when last light is turned off" etc., but that's probably out of scope.

Reply:
I think it's the behavior options that are hard to understand, not the behavior description.